webpay/pay/views.py

```diff
@@ -427,6 +428,13 @@ def trans_start_url(request):
         log.info('async call got payment URL {url} for trans {tr}'
                  .format(url=url, tr=trans))
         data['url'] = url
+
+    if trans and trans['status'] == constants.STATUS_ERRORED:
+        statsd.incr('purchase.payment_time.errored')
+        log.exception('Purchase configuration failed: {0} with status {1}'
+                      'with status {1}'.format(trans_id, trans['status']))
+        return system_error(request, code=getattr(msg, trans['status_reason']))
```

Comment:
It shouldn't be in a `STATUS_ERRORED` state without `status_reason` being set, but we don't enforce that in solitude yet, so yeah sure let's go to `UNEXPECTED_ERROR`.